.buildkite/pipeline_pr.py

```diff
@@ -42,7 +42,7 @@ def get_changed_files(branch):
 
 build_grp = group(
     "📦 Build",
-    "./tools/devtool -y test -- ../tests/integration_tests/build/",
+    "./tools/devtool -y test -- ../tests/integration_tests/build/ --ignore ../tests/integration_tests/build/test_kani.py",
```

Comment:
Are Kani tests suitable to be located under `tests/integration_tests/build/`? `functional` or `security` may be better places to have them? or kani specific directory?

Reply:
My thinking here was that `test_kani.py` is similar to `test_unittests.py`, which lives in `build`. As for its own directory, I felt it to be a bit awkward to have it with just a single file

Reply:
That makes sense. Thanks.

Reply:
I would rather go with the single file in a directory approach, or even just move it to `tests/integration_tests/test_kani.py`). Otherwise having to skip it looks even more awkward IO.

Reply:
Ah, it didnt cross my mind that we can just have individual files laying around. Fixed!